sample_apps/simple_app_under_keystone/instructions.md

```diff
@@ -8,12 +8,26 @@ No dependencies on anything, can build on any architecture and with any compiler
 
 ## Example App
 
+Start Qemu
+
+1. `docker pull keystoneenclaveorg/keystone:dev`
+2. `docker run -it --entrypoint /bin/bash keystoneenclaveorg/keystone:dev --volume ABS_CERTIFIER_PATH:/certifier`
```

Comment:
Let's explain what ABS_CERTIFIER_PATH is supposed to be

Reply:
resolved